.aicoe-ci.yaml

```diff
@@ -6,11 +6,11 @@ check:
   # - thoth-build
```

Comment:
Oh also can you uncomment this line.
Sorry forgot to mention it before

Reply:
:+1:  no worries, done.